mpop_service/main.cpp

```diff
@@ -8,19 +8,20 @@
 #include <QProcessEnvironment>
 #include "mpopservice.h"
 #include "config.h"
+#include "logutils.h"
 #include <QDebug>
 
 
 int main(int argc, char *argv[])
 {
     QCoreApplication application(argc, argv);
     application.setApplicationName("mpop_service");
+    logutils();
     Config config;
     MPopService::load_config_from_env_vars(config);
     MPopService service(config);
     QTimer mytimer;
     QObject::connect(&mytimer,&QTimer::timeout, std::bind(MPopService::timeWatcher, config));
-    mytimer.start(config.periodic_interval);
-
+    mytimer.start(config.periodic_interval);    
```

Comment:
This white space change doesn't match our coding style. (let's avoid trailing spaces)